docs/getting-started/installation-common/unified-installer.rst

```diff
@@ -19,7 +24,81 @@ Note that if you're on CentOS 7, only root offline installation is supported.
 Download and Install
 -----------------------
 
-For installation without root privileges, follow the instructions on `Scylla Download Center <https://www.scylladb.com/download/?platform=tar>`_.
+#. Download the latest tar.gz file for ScyllaDB |SCYLLADB_VERSION| (x86 or ARM) from https://downloads.scylladb.com/downloads/scylla/relocatable/scylladb-5.2/.
+#. Uncompress the downloaded package.
+
+   The following example shows the package for ScyllaDB 5.2.4 (x86):
+
+   .. code:: console
+
+    tar xvfz tar scylla-unified-5.2.4-0.20230623.cebbf6c5df2b.x86_64.tar.gz
+
+#. Install OpenJDK 8 or 11.
+
+   The following example shows Java installation on a CentOS-like system:
+
+   .. code:: console
+    
+    sudo yum install -y java-1.8.0-openjdk
```

Comment:
I think we should change it to `OpenJDK 11` rather to `8`

Reply:
You're right. I've made this change for packages, but not here. Fixing.